web/admin/src/Admin.js

```diff
@@ -83,19 +82,26 @@ export default class Admin extends PureComponent {
   componentDidUpdate() {
     if (!this.state.sessionId) {
       const sessionIds = Object.keys(this.state.sessions)
-      if (sessionIds.length) this.setState({ sessionId: sessionIds[0] })
+      if (sessionIds.length) {
+        this.setState({ sessionId: sessionIds[0] })
+      }
+    }
+  }
+
+  checkSessionStatus = () => {
```

Comment:
To be more specific about what this function returns, maybe name it the more-specific `isSessionEnded` and reverse the true/false